pkg/api/deploymentapi/eskeystoreapi/get.go

```diff
@@ -0,0 +1,75 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package eskeystoreapi
+
+import (
+	"errors"
+
+	"github.com/elastic/cloud-sdk-go/pkg/api"
+	"github.com/elastic/cloud-sdk-go/pkg/api/apierror"
+	"github.com/elastic/cloud-sdk-go/pkg/client/deployments"
+	"github.com/elastic/cloud-sdk-go/pkg/models"
+	"github.com/elastic/cloud-sdk-go/pkg/multierror"
+)
+
+// GetParams is consumed by the Get function.
+type GetParams struct {
+	*api.API
+
+	DeploymentID string
+	RefID        string
+}
+
+// Validate ensures the parameters are usable by Get.
+func (params GetParams) Validate() error {
+	var merr = multierror.NewPrefixed("invalid elasticsearch keystore get params")
+
+	if params.API == nil {
+		merr = merr.Append(apierror.ErrMissingAPI)
+	}
+
+	if len(params.DeploymentID) != 32 {
+		merr = merr.Append(apierror.ErrDeploymentID)
+	}
+
+	if params.RefID == "" {
```

Comment:
Does refID autodiscovery not make sense for this API?

Reply:
d'uh it does. I completely missed that 🤦